datasafe-simple-adapter/datasafe-simple-adapter-impl/src/main/java/de/adorsys/datasafe/simple/adapter/impl/SimpleDatasafeServiceImpl.java

```diff
@@ -226,25 +226,30 @@ private static SystemRootAndStorageService useAmazonS3(AmazonS3DFSCredentials df
 
         log.info(lsf.toString());
         AmazonS3ClientBuilder amazonS3ClientBuilder = AmazonS3ClientBuilder.standard()
-                .withCredentials(
-                        new AWSStaticCredentialsProvider(
-                                new BasicAWSCredentials(
-                                        amazonS3DFSCredentials.getAccessKey(),
-                                        amazonS3DFSCredentials.getSecretKey()))
-                )
-                .enablePathStyleAccess();
+            .withCredentials(
+                new AWSStaticCredentialsProvider(
+                    new BasicAWSCredentials(
+                        amazonS3DFSCredentials.getAccessKey(),
+                        amazonS3DFSCredentials.getSecretKey()))
+            );
 
-        boolean useEndpoint = !amazonS3DFSCredentials.getUrl().equals(AMAZON_URL)
-                && !amazonS3DFSCredentials.getUrl().startsWith(S3_PREFIX);
+        boolean useEndpoint = !amazonS3DFSCredentials.getUrl().matches(AMAZON_URL)
+            && !amazonS3DFSCredentials.getUrl().startsWith(S3_PREFIX);
+        lsf = new LogStringFrame();
```

Comment:
Maybe we can use simple `log.info` call instead of `LogStringFrame` as `LogStringFrame` is more for printing configuration and not couple of lines

Reply:
As these frames are only logged once and are very important I dont want to change this. Its important for Timp Team and Devops. And it is a kind of configuration.